tests/baselines/reference/jsdocTypeTagCast.types

```diff
@@ -209,4 +209,18 @@ if(/** @type {numOrStr is string} */(numOrStr === undefined)) { // Error
 }
 
 
+var asConst1 = /** @type {const} */(1);
+>asConst1 : 1
+>(1) : 1
+>1 : 1
 
+var asConst2 = /** @type {const} */({
+>asConst2 : { x: number; }

```

Comment:
This isn't the expected `as const` result of this expression? It should be `{readonly x: 1}`.

Reply:
Yeah, it looks like I missed a required change to `isConstContext`, will update shortly.